### DISCUSSION TITLE
how to use laravel livewire component in package without register it in service provider

### DISCUSSION THREAD

**Discussion by vahidalvandi** [livewire/livewire#5523] [1. Help]
i have many livewire component and need to add in service provider .I think that the large number of components will reduce the performance of the application in general . For this reason, I want to call the components inside the packages in the blades in such a way that there is no need to define them in the service provider.

**in service provider**
`        Livewire::component('filter', Filter::class);`

**in blade**
`    @livewire('filter')`

**i want to call livewire component without register in serivce provider in blade like this**
` Livewire::component('filter', Filter::class)->run();`

**Comment by joshhanley** [comment]
@vahidalvandi I'm not sure if it will work or not, but you could try using `@livewire(/Path/To/Package/Component::class)`.

Hope this helps!

**Reply by vahidalvandi** [reply]
thank you i will test it

**Reply by joshhanley** [reply]
No worries! 🙂
